.github/workflows/build-and-test-other.yaml

```diff
@@ -22,17 +22,22 @@ jobs:
 
         include:
         - arch: "armel"
-          distro: "wheezy"
-          base_url: "http://archive.debian.org/debian"
-          cflags: "-std=gnu99 -O2 -mthumb -mthumb-interwork -march=armv4t"
+          distro: "stretch"
+          base_url: "http://deb.debian.org/debian/"
+          cflags: "-O2 -mthumb -mthumb-interwork -march=armv4t"
+          additional_packages: |
+            http://ftp.us.debian.org/debian/pool/main/c/cmake/cmake-data_3.13.2-1~bpo9+1_all.deb \
```

Comment:
This list of hardcoded deb URLs look weird, I'd like to look for any other cleaner alternative. I'll be back on this topic soon.

Reply:
Please be my guest. I spent quite some time looking at debootstrap. What we really want is cmake 3.13, the others are dependencies.